frontend/public/components/monitoring.tsx

```diff
@@ -1834,31 +1836,37 @@ type Silences = {
   loadError?: string;
 };
 
-type PrometheusAlert = {
+export type PrometheusAlert = {
   activeAt?: string;
   annotations: PrometheusLabels;
   labels: PrometheusLabels & {
     alertname: string;
   };
-  state: AlertStates;
-  value?: number;
+  state: AlertStates | string;
+  value?: number | string;
```

Comment:
I think this should stay as just `value?: number`. Although `string` is technically possible, it's not actually used (see https://prometheus.io/docs/prometheus/latest/querying/basics/#expression-language-data-types).

Reply:
In Thanos alerts ```value``` is of type string.

Reply:
In Prometheus, they are numbers represented as strings, but the frontend code treats them as `number`. I assume Thanos is the same, so I'm still thinking this should be just `number`.

I want to be careful about changing these types since they define an interface between the dev perspective, the admin perspective, Prometheus and Thanos.